test/commands/crashes/upload-symbols/upload-symbols-test.ts

```diff
@@ -69,10 +69,28 @@ describe("upload-symbols command", () => {
     abortSymbolUploadSpy = Sinon.spy();
   });
 
-  describe("when network requests are successful", () => {
+  describe("when Breakpad network requests are successfu", () => {
```

Comment:
Small typo in `describe` text (missing `l` off the end)

Reply:
good catch! i don't know how that snuck in there.